config/software/automake.rb

```diff
@@ -24,9 +24,8 @@
 license_file "COPYING"
 skip_transitive_dependency_licensing true
 
-version("1.16") { source sha256: "80da43bb5665596ee389e6d8b64b4f122ea4b92a685b1dbd813cd1f0e0c2d83f" }
-version("1.15") { source sha256: "7946e945a96e28152ba5a6beb0625ca715c6e32ac55f2e353ef54def0c8ed924" }
-version("1.11.2") { source sha256: "c339e3871d6595620760725da61de02cf1c293af8a05b14592d6587ac39ce546" }
+version("1.16.3") { source sha256: "ce010788b51f64511a1e9bb2a1ec626037c6d0e7ede32c1c103611b9d3cba65f" }
```

Comment:
Can we add the `# version_list:` comment inline here?

Reply:
yes , I'll update PR accordingly